### PR TITLE
Minor tag changes

### PR DESCRIPTION
**Pull request by jperezgonzal**
## Description
Se hicieron cambios menores reportado por UX
- Se cambió el color del texto default
- Se eliminó el type "Neutral"
- Se modifico el margen del icono dismiss
- Se agregó un stroke a la imagen (left content)

## Checks
Are you sure you followed all those steps? In such case, let's say with me "I solemnly declare that ...":
- [x] I have coded an example inside the Demo App,
- [x] I have added proper tests,
- [ ] I have modified the [Changelog](https://github.com/mercadolibre/fury_andesui-android/blob/develop/CHANGELOG.md) file,
- [ ] I have updated GitHub wiki,
- [ ] I have the explicit approval of one or more members of the UX Team,
- [ ] I have updated the version in gradle.properties file.

## Change type
This is easy, let us know what this code is about:
- [ ] This code adds a new component
- [x] This code includes improvements to an existent component
- [ ] This code improves or modifies ONLY the Demo App.

## Check common errors
Whether you are an Android or iOS developer and if you're still there then we'll give you a present:
https://proandroiddev.com/how-to-maximize-androids-ui-reusability-5-common-mistakes-cb2571216a9f

### CODE REVIEW

**Comment by furyGorgoryAdmin** [comment]
### Informative Menu <h2>  ### 

### Use any of these tags in a pull request description or leave a comment ### 
| Tag | Description |
|---|---|
| **[canary]**| Creates a build that can only be deployed with canary strategy. | 
| **[build]** |  Creates version or build from a branch that has been tagged as "no releasable". | 
| **[name:version-name]** |Specify the name of the build. For example: [build] [name:1.2.3-name] or [canary] [name:1.2.3-name].|
| **[major]** | To automatically increase the first position of the version. For example: [build] [major] or [canary] [major].|
| **[minor]** | To automatically increase the second position of the version. For example: [build] [minor] or [canary] [minor]. |
| **[patch]** | To automatically increase the third position of the version. For example: [build] [patch] or [canary] [patch].|  

[For more information about ci-cd](http://docs.furycloud.io/?lang=esp&section=businessassurance)
[For more information about semver](https://semver.org/#summary)

File: components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt [hunk 1]
```diff
@@ -7,27 +7,40 @@ import android.widget.ImageView
...
     abstract fun size(context: Context): Int
     abstract fun border(context: Context): Float
     abstract fun view(context: Context, color: AndesColor, rightContent: RightContent, callback: View.OnClickListener): View
 }
 
 internal object AndesTagRightContentNone : AndesTagRightContentInterface() {
-    override fun leftMargin(context: Context): Int = 0
-    override fun rightMargin(context: Context): Int = 0
+    override fun leftMargin(context: Context, size: AndesTagSize): Int = 0
+    override fun rightMargin(context: Context, size: AndesTagSize): Int = 0
```
[the remark this answers is not in the document]
Comment (jperezgonzal) [reply components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt:23]: El const lo tendria que poner en la clase publica (AndesTagSimple).
Pero como esto esta en la interfaz del enum el const tiene que ser publico en lugar de privado. Por lo que los dev podrian ver esa const.

File: components/src/main/java/com/mercadolibre/android/andesui/tag/factory/AndesTagSimpleConfigurationFactory.kt [hunk 2]
```diff
@@ -9,9 +9,10 @@ import com.mercadolibre.android.andesui.tag.type.AndesSimpleTagTypeInterface
 
 internal data class AndesTagSimpleConfiguration(
     val text: String? = null,
+    val backgroundColor: AndesColor? = null,
```
[the remark this answers is not in the document]
Comment (jperezgonzal) [reply components/src/main/java/com/mercadolibre/android/andesui/tag/factory/AndesTagSimpleConfigurationFactory.kt:12]: Ahi hice el cambio para que el background no sea nulo. Tanto esa variable como la de dismissColor es de uso interno y se setan segun el enum "type". 
Las pruebas ya estaban y serían por ejemplo que se prueben todos los tipos de tag posibles.

File: components/src/main/java/com/mercadolibre/android/andesui/tag/AndesTagSimple.kt [hunk 3]
```diff
@@ -225,14 +222,15 @@ class AndesTagSimple : ConstraintLayout {
...
-                constraintSet.setMargin(R.id.simple_tag_text, ConstraintSet.END, config.rightContent.content.leftMargin(context))
+                constraintSet.setMargin(R.id.simple_tag_text, ConstraintSet.END, config.rightContent.content.leftMargin(context, size))
             }
             constraintSet.applyTo(containerTag)
         }
     }
 
     private fun setupLeftContent(config: AndesTagSimpleConfiguration) {
         if (config.leftContent != null && config.leftContentData != null && config.leftContent != AndesTagLeftContent.NONE) {
+            left_content.removeAllViews()
```
Comment (jiurchuk) [inline components/src/main/java/com/mercadolibre/android/andesui/tag/AndesTagSimple.kt:233]: Por qué usamos snake case ?

File: components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (jiurchuk) [reply components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt:23]: Pero no podemos tener un companion object dentro de este object?

File: components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt [hunk 4]
```diff
@@ -7,27 +7,40 @@ import android.widget.ImageView
...
     override fun size(context: Context): Int = 0
     override fun border(context: Context): Float = 0f
     override fun view(context: Context, color: AndesColor, rightContent: RightContent, callback: View.OnClickListener): View = View(context)
 }
 
 internal object AndesTagRightContentDismiss : AndesTagRightContentInterface() {
-    override fun leftMargin(context: Context): Int = context.resources.getDimension(R.dimen.andes_tag_medium_margin).toInt()
-    override fun rightMargin(context: Context): Int = context.resources.getDimension(R.dimen.andes_tag_medium_margin).toInt()
+    override fun leftMargin(context: Context, size: AndesTagSize): Int {
+        return if (size == AndesTagSize.SMALL) {
```
Comment (jiurchuk) [inline components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt:31]: Podemos usar `when` en vez de if?

File: components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt [hunk 5]
```diff
@@ -7,27 +7,40 @@ import android.widget.ImageView
...
+            context.resources.getDimension(R.dimen.andes_tag_small_margin).toInt()
+        } else {
+            context.resources.getDimension(R.dimen.andes_tag_medium_margin).toInt()
+        }
+    }
+    override fun rightMargin(context: Context, size: AndesTagSize): Int {
+        return if (size == AndesTagSize.SMALL) {
+            context.resources.getDimension(R.dimen.andes_tag_small_margin).toInt()
+        } else {
+            context.resources.getDimension(R.dimen.andes_tag_medium_margin).toInt()
```
Comment (jiurchuk) [inline components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt:41]: Idem `when`

File: components/src/test/java/com/mercadolibre/android/andesui/tag/AndesTagSimpleTest.kt [hunk 6]
```diff
@@ -106,16 +92,9 @@ class AndesTagSimpleTest {
 
     @Test
     fun `Simple, Small, Default border color`() {
-        attrs = AndesTagSimpleAttrs(AndesTagType.DEFAULT, AndesTagSize.SMALL, "Body")
-        val config = configFactory.create(attrs)
-        assertEquals(config.borderColor, R.color.andes_gray_450_solid.toAndesColor())
-    }
```
Comment (jiurchuk) [inline components/src/test/java/com/mercadolibre/android/andesui/tag/AndesTagSimpleTest.kt:112]: Estos test que sacamos, los estamos agregando en otro lado?

File: components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (jperezgonzal) [reply components/src/main/java/com/mercadolibre/android/andesui/tag/rightcontent/AndesTagRightContentInterface.kt:23]: @jiurchuk es una interfaz entonces no podemos tener un companion object, al no ser que creemos una clase ahi mismo y que esa clase tenga el enum

**Pull request merged**